python/graphstorm/sagemaker/run/sagemaker_infer.py

```diff
@@ -0,0 +1,318 @@
+""" Copyright 2023 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Inference entry point.
+
+    As SageMaker only accept a single script as the entry point.
+    We have to put all code in one file.
+"""
+# Install additional requirements
+import os
+import socket
+import time
+import json
+import subprocess
+import argparse
+from threading import Thread, Event
+import sys
+import queue
+
+import boto3
+from graphstorm.config.config import SUPPORTED_TASKS
+from graphstorm.config.config import BUILTIN_TASK_NODE_CLASSIFICATION
+from graphstorm.config.config import BUILTIN_TASK_NODE_REGRESSION
+from graphstorm.config.config import BUILTIN_TASK_EDGE_CLASSIFICATION
+from graphstorm.config.config import BUILTIN_TASK_EDGE_REGRESSION
+from graphstorm.config.config import BUILTIN_TASK_LINK_PREDICTION
```

Comment:
Use parentheses for multiple imports ([PEP328](https://peps.python.org/pep-0328/)) e.g.

```
from graphstorm.config.config import (SUPPORTED_TASKS,
    BUILTIN_TASK_NODE_CLASSIFICATION,
    BUILTIN_TASK_NODE_REGRESSION,
    ...
```

Same for all other places multiple imports are done from the same package.

Reply:
This comment is not resolved right @classicsong ?